app/main.py

```diff
@@ -1,3 +1,44 @@
+from typing import TypeVar, Union
+
+Self = TypeVar("Self", bound="Distance")
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
```

Comment:
i used only `float` because PEP 484 specifies the following:
> Rather than requiring that users write import numbers and then use numbers.Float etc., this PEP proposes a straightforward shortcut that is almost as effective: when an argument is annotated as having type float, an argument of type int is acceptable; similar, for an argument annotated as having type complex, arguments of type float or int are acceptable.

can i keep this type annotation?

Reply:
but remember, explicit is better than implicit)

Reply:
agree)